Src/Plugins/Tests.Security/TestData/UseSecureApi/ExpectedOutputs/SEC104.Memory.Allocation_malloca.sarif

```diff
@@ -84,11 +84,11 @@
             }
           ],
           "fingerprints": {
-            "SecretHashSha256/current": "a4965ae12f3568daff154d92dc41cddff82ae7b6674138279bbdac60217d7d1b",
-            "AssetFingerprint/current": "{\"id\":\"_malloca\"}",
-            "ValidationFingerprintHashSha256/current": "582290aa80b7a22d4aa98aa1227346b61831fcdc156e2a570bc6ad5b8763ccf2",
-            "SecretFingerprint/current": "{}",
-            "ValidationFingerprint/current": "{\"id\":\"_malloca\"}"
+            "SecretHashSha256/v0": "a4965ae12f3568daff154d92dc41cddff82ae7b6674138279bbdac60217d7d1b",
```

Comment:
I realize this wouldn't be part of your change, but it would be nice if the SDK somehow parsed and exposed the fingerprint versions. Maybe something like GetFingerprints(string key) that returned a list of versions and their values? It would prevent users from having to understand the versioning rules.